web/src/main/java/ua/com/vertex/context/SessionListener.java

```diff
@@ -0,0 +1,21 @@
+package ua.com.vertex.context;
+
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpSessionEvent;
+import javax.servlet.http.HttpSessionListener;
+
+@Component
+public class SessionListener implements HttpSessionListener {
+
+    @Override
+    public void sessionCreated(HttpSessionEvent event) {
+        System.out.println("sessionCreated");
+        event.getSession().setMaxInactiveInterval(30 * 60);
```

Comment:
In Spring Boot it can be set in application.properties. In usual Spring MVC a listener has to be created.